src/modules/Elsa.Studio.UIHints/Components/Dictionary.razor.cs

```diff
@@ -60,32 +60,46 @@ private IEnumerable<ExpressionDescriptor> GetSupportedExpressions()
         return ExpressionDescriptorProvider.ListDescriptors().Where(x => !_uiSyntaxes.Contains(x.Type) && x.IsBrowsable).ToList();
     }
 
-    private static string GetDefaultExpressionType()
+    private string GetDefaultExpressionType()
     {
-        // Default to Literal for new entries to avoid unnecessary wrapping
-        return "Literal";
+        var defaultExpressionType = GetSupportedExpressions().FirstOrDefault()?.Type ?? "Literal";
```

Comment:
What's the rationale behind this change? Given that the support expressions list can vary from installation to installation, as well as the order, the default expression type now becomes non-deterministic.

Reply:
Ok, in that case it should be OK, given that we're not aware of any issues with the Cases component. Should you happen to understand the logic behind it, it would be good to add a comment for our future selves.